superset-frontend/src/dashboard/components/DashboardBuilder/DashboardContainer.tsx

```diff
@@ -91,9 +91,9 @@ const DashboardContainer: FC<DashboardContainerProps> = ({ topLevelTabs }) => {
       const { scope } = filterScope;
       const chartsInScope: number[] = getChartIdsInFilterScope({
         filterScope: {
-          scope: scope.rootPath,
+          scope: scope?.rootPath,
```

Comment:
In what cases `scope` field could be optional? I thought all filter has default "all" scope and [] for immune.